src/components/CardStatus/CardStatus.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <ClientOnly placeholder="...">
+    <p
+      class="card-status"
+      :class="{ 'card-status--open': isOpen }"
+      v-bind="$attrs"
+    >
+      <template v-if="isOpen">
+        {{ $t("open") }}
+        <SvgIcon name="location-open-icon" class="card-status__icon" />
+      </template>
+      <template v-else>
+        {{ $t("closed") }}
+        <SvgIcon name="location-closed-icon" class="card-status__icon" />
+      </template>
+    </p>
+  </ClientOnly>
+</template>
+
+<script lang="ts">
+export default {
+  inheritAttrs: false,
+};
+</script>
+
+<script setup lang="ts">
+import type { OpeningHours } from "~/types/OpeningHours";
+
+const props = defineProps<{ openingHours: OpeningHours[] }>();
+const isOpen = computed(() => {
+  const indexToday = 0;
+  const openingHoursToday = props.openingHours[indexToday].time; //TODO: say what
```

Comment:
what does this comment mean?

Reply:
at that moment I was fascinated by the fact that we rely on deploys to determine what "today" means.
I guess I'll remove the comment and we think later if we need to change something here :)